AdyenCardScanner/Sources/CardScanner.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+protocol CardScanning {
+    static func createCardScanner(
+        completion: @escaping (Result<CreditCard, CardScannerError>) -> Void
+    ) -> UIViewController?
+
+    static var isAvailable: Bool { get }
+}
+
+public enum CardScanner: CardScanning {
+
+    // MARK: - Properties
+
+    private static let cardScannerAssembler = CardScannerAssembler()
```

Comment:
I know it's not injected but is it supposed to be `CardScannerAssembling`?

Reply:
Yeah, it is but since it cannot be injected, I just leave it like this. You see any issues with it like this?